nextcloud/Readme.md

```diff
@@ -0,0 +1,19 @@
+This directory contains the terraform and heat scripts for running Nextcloud.
+
```

Comment:
Needs short description of what the nextcloud is a link to the upstream project.

Reply:
I don't know what you mean by this, as in a link to the catalyst cloud documentation

Reply:
No, what nextcloud is and a link to the nextcloud project.
Something like:

*[Nextcloud](https://nextcloud.com/) is an online collaboration platform*

Reply:
Should I not leave that for the catalyst cloud documentation? I have something similar written in the deploying-nextcloud.rst file but I haven't figured out how to do links in text yet

Reply:
Just drop a small single-line description for people who arrive in the repository through google and not through documentation project.